L1Trigger/L1TGlobal/plugins/L1TGlobalPrescalesVetosFractESProducer.cc

```diff
@@ -86,7 +86,8 @@ L1TGlobalPrescalesVetosFractESProducer::L1TGlobalPrescalesVetosFractESProducer(c
   setWhatProduced(this);
   //setWhatProduced(this, conf.getParameter<std::string>("label"));
 
-  m_numberPhysTriggers = 512;
+  //m_numberPhysTriggers = 512;
+  m_numberPhysTriggers = 513;
```

Comment:
i think this has to stay at 512, since this is the max number of possible L1 seeds in the GT